psa-crypto-sys/src/lib.rs

```diff
@@ -39,9 +39,9 @@ pub use types::*;
 
 #[cfg(feature = "implementation-defined")]
 pub use psa_crypto_binding::{
-    psa_close_key, psa_crypto_init, psa_destroy_key, psa_export_public_key, psa_generate_key,
-    psa_import_key, psa_key_attributes_t, psa_open_key, psa_reset_key_attributes, psa_sign_hash,
-    psa_verify_hash,
+    mbedtls_psa_crypto_free, psa_close_key, psa_crypto_init, psa_destroy_key,
```

Comment:
Same for `mbedtls_psa_crypto_free`, I don't think we should declare it here.

Reply:
I'm not sure how we should deal with this. For `psa-crypto` to be portable to any PSA Crypto implementation, we can't have this `mbedtls` specific method declared here, but I'm not sure how we remove it from `psa-crypto` and allow Parsec to call it without binding straight through to `mbedtls`.

Reply:
I think this is is used inside Mbed Crypto for testing but should not be required in this crate. If it is, and creating leaks in our applications, we should probably discuss this at a PSA Crypto level.